enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk_setup/components/ColorCustomizationSection.tsx

```diff
@@ -57,7 +57,7 @@ export const ColorCustomizationSection = ({
         )}
       </Group>
 
-      <Group align="start" gap="xl" mb="lg">
+      <Group align="start" gap="xl">
```

Comment:
Fixes a weird padding that was below the appearance settings if there were no settings below the color pickers.

<img width="628" height="532" alt="CleanShot 2568-08-20 at 04 25 24@2x" src="https://github.com/user-attachments/assets/0a95e32d-e28b-4d21-9a21-f67d36b7d2ff" />